index.html

```diff
@@ -1816,37 +1816,39 @@ <h3>Lifecycle Details</h3>
 
         <ol>
           <li>
-The <a>verifier</a> defines its policies for accepting
+A <a>verifier</a> defines its own policies for accepting
 <a>verifiable credentials</a> from <a>holders</a> for its supported actions.
           </li>
           <li>
-The <a>issuer</a> issues a <a>verifiable credential</a> to the <a>holder</a>.
+An <a>issuer</a> issues one or more <a>verifiable credentials</a> to 
+a <a>holder</a>.
           </li>
           <li>
-The <a>holder</a> might pass on its <a>verifiable credentials</a> to another
-<a>holder</a>.
+A <a>holder</a> might pass on one or more of its <a>verifiable credentials</a> 
+to another <a>holder</a>.
           </li>
           <li>
-The final <a>holder</a> presents its <a>verifiable credentials</a> to the
-<a>verifier</a> in a <a>verifiable presentation</a>, requesting a supported
+A <a>holder</a> presents one or more of its <a>verifiable credential(s)</a> 
+to a <a>verifier</a> in a <a>verifiable presentation</a>, requesting a supported
 action.
           </li>
           <li>
-The <a>verifier</a> <a>verifies</a> the authenticity of the
-<a>verifiable presentation</a> and <a>verifiable credentials</a>.
+A <a>verifier</a> <a>verifies</a> the authenticity of a <a>verifiable presentation</a> 
+and the <a>verifiable credential(s)</a> contained therein.
           </li>
           <li>
-The <a>verifier</a> <a>verifies</a> that the <a>holder</a> possesses the
-<a>verifiable credentials</a>.
+A <a>verifier</a> <a>verifies</a> that the <a>holder</a> that made the 
+<a>verifiable presentation</a> possesses the <a>verifiable credential(s)</a>
+contained therein.
           </li>
           <li>
-The <a>verifier</a> decides whether to accept the <a>verifiable credentials</a>
-for the requested action, taking into account its policy, the <a>holder</a>,
-and the contents of the <a>verifiable credentials</a>.
+A <a>verifier</a> decides whether to accept the <a>verifiable credentials</a> 
```

Comment:
Similarly this should be The verifier for the reasons stated above. (Note you are already talking about The verifiable credentials :-)